en/docs/install-and-setup/configuration-for-wso2-api-manager.md

```diff
@@ -66,4 +76,54 @@ Follow the instructions below to configure the WSO2 Microgateway Toolkit and th
         validateSubscription = false
```

Comment:
Change the indentation inside the block as below
```
    ```tab="Example"
    [[jwtTokenConfig]]
        issuer="https://localhost:9443/oauth2/token"
        audience="http://org.wso2.apimgt/gateway"
        certificateAlias="wso2carbon2"
        validateSubscription = false
    ```
```